SPECS/python3/python3.spec

```diff
@@ -160,11 +158,17 @@ chmod -v 755 %{buildroot}%{_libdir}/libpython3.7m.so.1.0
 %{_fixperms} %{buildroot}/*
 ln -sf libpython3.7m.so %{buildroot}%{_libdir}/libpython3.7.so
 
+# Install pathfix.py to bindir
+cp -p Tools/scripts/pathfix.py %{buildroot}%{_bindir}/pathfix3.7.py
+
 # Remove unused stuff
 find %{buildroot}%{_libdir} -name '*.pyc' -delete
 find %{buildroot}%{_libdir} -name '*.pyo' -delete
 find %{buildroot}%{_libdir} -name '*.o' -delete
 rm %{buildroot}%{_bindir}/2to3
+rm -rf %{buildroot}%{_bindir}/__pycache__
+
+ln -s ./pathfix3.7.py %{buildroot}%{_bindir}/pathfix.py
```

Comment:
2 things:
1. Why don't we link to the newly-copied version (e.g. use the same path from the copy above)? Right now it looks like we're copying to one place and linking a (potentially different) file from elsewhere even though they have the same name. It looks suspicious to say the least.:)
1. Let's move this directly next to where you copy `pathfix.3.7.py`. It looks like it's related closer to the `Install pathfix.py to bindir` comment, than to `Remove unused stuff`.

Reply:
1. Since the versioned (`pathfix3.7.py`) and unversioned (`pathfix.py`) are installed to the same directory, a relative symlink is made.
2. Agreed, moved.